scripts/dnt.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  dirname,
+  fromFileUrl,
+  join,
+} from "https://deno.land/std@0.147.0/path/mod.ts";
+
+import {
+  build,
+  emptyDir,
+} from "https://raw.githubusercontent.com/roj1512/dnt/main/mod.ts";
+
+import package_ from "./package.json" assert { type: "json" };
+
+const version = Deno.args[0];
+if (!version) {
+  throw new Error("Provide the version as an argument");
+}
+
+const rootDir = join(dirname(fromFileUrl(import.meta.url)), "../");
+const outDir = join(
+  rootDir,
+  "out",
+);
+
+await emptyDir(outDir);
+
+await build({
+  outDir,
+  shims: {
+    deno: true,
+  },
+  package: {
+    version,
+    ...package_,
+  },
+  esModule: false,
+  entryPoints: ["./src/mod.ts"],
+  mappings: {
+    "https://deno.land/x/better_fluent@v0.1.0/mod.ts": {
+      name: "@moebius/fluent",
+      version: "^1.0.0",
+    },
+    "https://lib.deno.dev/x/grammy@1.x/mod.ts": {
+      name: "grammy",
+      version: "^1.9.0",
+      peerDependency: true,
+    },
+    "https://lib.deno.dev/x/grammy@1.x/core/client.ts": {
```

Comment:
I am not proud of it. But its for the testing library we used: https://github.com/dcdunkan/tests. In that, internal stuffs are [imported](https://github.com/dcdunkan/tests/blob/7aeba5bb93185def5966482d574a896abb5d5895/deps.ts#L7) from this URL. So, if this import does not have a Node mapping, it will try to build with the Deno (lib.deno.dev) version of grammY and throw (**Only if tests are enabled in dnt options**):

```
[dnt] Top level await cannot be used when distributing CommonJS/UMD (See deps/deno.land/x/grammy@v1.9.0/platform.deno.ts 21:17). Please re-organize your code to not use a top level await or only distribute an ES module by setting the 'scriptModule' build option to false.
```

So, it was a required mapping. As I have mentioned earlier, I am ready to remove tests related to this, if its hard to maintain.

Reply:
Oh. Didn't know about this. Thanks. I'll update the library accordingly.

Reply:
I guess this has been resolved then.